xml/slert_usingshortcuts.xml

```diff
@@ -178,9 +178,9 @@
 
   <para>
    The <command>set</command> subcommand has a limited number of shortcuts.
-   Basically, the <option>-s</option>/<option>--set</option> option is optional in most cases
-   and the <option>-l/</option>/<option>--list</option> option is also optional if you want to
-   list sets. For example, these commands are equivalent.
+   The options <option>-s</option>/<option>--set</option> are optional usually
```

Comment:
I used only `--set` and `--list` as in my previous example to avoid the singular/plural mess.